public/admin/config.yml

```diff
@@ -255,3 +255,86 @@ collections:
         file: src/cms-content/learn/footer.json
         fields:
           - { label: Body, name: body, widget: markdown, default: true }
+      - name: Case Studies
+        file: 'src/cms-content/learn/case-studies.json'
+        label: Case Studies
+        fields:
+          - label: Title
+            name: title
+            widget: text
+            hint: Title for the case studies landing page
+          - label: Intro
+            name: intro
+            widget: markdown
+            hint: Introduction for the case studies landing page
+          - label: Categories
+            name: categories
+            widget: list
+            fields:
+              - {
+                  label: Category Name,
+                  name: title,
+                  widget: text,
+                  hint: Name of the category,
+                }
+              - {
+                  label: Anchor Link ID,
```

Comment:
I think just 'Category ID' would be good here (in line with term-id, etc.), rather than using 'anchor'/eng-language